cmds/vpdbootmanager/addentry/netboot.go

```diff
@@ -0,0 +1,56 @@
+package addentry
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/booter"
+	"github.com/systemboot/systemboot/pkg/vpd"
+)
+
+func addNetbootEntry() {
+	if len(os.Args) < 5 {
+		fmt.Println(usage)
+		os.Exit(1)
```

Comment:
return an error here instead of exiting - the caller can decide whether this is fatal or not